docs/install/grafana-setup.rst

```diff
@@ -79,6 +79,14 @@ directory to begin.
    $ sudo docker-compose build
    $ sudo docker-compose up -d
 
+.. note::
+   In the event docker container build fails to verify certificates, try to disable 
+   network proxy services as a first step.
+   Network proxy services running on the host can significantly interfere with 
+   OpenSSL's ability to retrieve a proper certificate chain when accessing websites 
+   from within Docker containers.
+   
```

Comment:
```suggestion
.. note::

   To troubleshoot Docker container build failures related to certificate verification, try 
   disabling any network proxy services on the host system. These proxy services can interfere 
   with OpenSSL's ability to retrieve a correct certificate chain when the container accesses 
   external websites.
```
Maybe something like this would be cleaner?